data-tool/notebooks/corps_onboarding_process_flow/migration_status_tracking.ipynb

```diff
@@ -89,15 +87,15 @@
     "    \"batch_size\": \"Batch Size\",\n",
     "    \"migrated_businesses\": \"Migrated Businesses\",\n",
     "    \"notes\": \"Notes\"\n",
-    "}\t\t\t\t\t\n",
+    "}\n",
     "\n",
     "TAB_NAMES = {\n",
     "    \"status\": \"Migration Status\",\n",
     "    \"summary\": \"Batch Summary\"\n",
     "}\n",
     "\n",
     "CONFIG = {\n",
-    "    'batch_size': 5000,\n",
+    "    'batch_size': 1000,\n",
```

Comment:
Oracle has a limit of 1000 for IN clauses.  This fixed an issue where all COLIN columns in spreadsheet were returning as N even when it should have been Y